From 8b6b9e2927c1191c8a896424b25653a001ee41ad Mon Sep 17 00:00:00 2001
From: Brian May <brian@microcomaustralia.com.au>
Date: Sun, 11 Oct 2015 12:12:44 +1100
Subject: Added a dummy class for HTMLParserError.

Fix problem introduced by python-3.5 and causes FTBFS for several Django
related packages.

Backport of upstream fix.
https://github.com/django/django/commit/b07aa52e8a8e4c7fdc7265f75ce2e7992e657ae9

Closes: #796104
---
 django/utils/html_parser.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/django/utils/html_parser.py b/django/utils/html_parser.py
index efe5422..ab1f654 100644
--- a/django/utils/html_parser.py
+++ b/django/utils/html_parser.py
@@ -9,7 +9,12 @@ use_workaround = (
     (current_version >= (3, 0) and current_version < (3, 2, 3))
 )
 
-HTMLParseError = _html_parser.HTMLParseError
+try:
+    HTMLParseError = _html_parser.HTMLParseError
+except AttributeError:
+    # create a dummy class for Python 3.5+ where it's been removed
+    class HTMLParseError(Exception):
+        pass
 
 if not use_workaround:
     if current_version >= (3, 4):
